frontend/src/components/LoxotronCasinoButtonComponent/index.tsx

```diff
@@ -0,0 +1,13 @@
+import React from "react";
+
+interface ButtonProps {
+    text: string;
+}
+
+const LoxotronCasinoButtonComponent: React.FC<ButtonProps> = function (props: ButtonProps) {
+    return (
+        <button>{props.text}</button>
+    )
+}
+
+export default LoxotronCasinoButtonComponent;
```

Comment:
Please, add an extension (or configure it in settings if possible) to your code editor to automatically add a blank new line at the end of the file